instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/AgentVersion.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.instrumentation.api;
+
+public class AgentVersion {
```

Comment:
since this is used by library instrumentation also, maybe rename to `Version` or maybe `InstrumentationApiVersion`?

Reply:
`Version` is too generic for me. And I am not sure about `Instrumentation*Api*Version`. I see it more like implementation version, the id of the specific artifact.

Reply:
I'm good with `InstrumentationVersion` (just trying to get `Agent` out of the name)

Reply:
renamed